CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## master (unreleased)
 
 ### New features
+* [#8055](https://github.com/rubocop-hq/rubocop/pull/8055): Add new `Style/RedundantRegexpCharacterClass` cop. ([@owst][])
```

Comment:
Can you add a blank line after the heading and remove the blank line between this and the next changelog entry?